rezerwacje-app/src/main/java/org/niewidoczniakademicy/rezerwacje/service/repository/FacultyRepository.java

```diff
@@ -8,6 +8,6 @@
 import java.util.Set;
 
 public interface FacultyRepository extends JpaRepository<Faculty, Long> {
-    Optional<Faculty> findFacultyByName(String name);
+    Optional<Faculty> findFacultyById(Long id);
```

Comment:
Do we need this method in repository? I'm pretty sure that there is one already one that returns Optional.

Reply:
Done